packages/backend/src/api/controllers/status/discovery/props/dashboardContracts/getIgnoreInWatchMode.ts

```diff
@@ -19,6 +20,7 @@ export function getIgnoreInWatchMode(
         return {
           name: getFieldName(field, viewABI),
           values: getValues(discovery, contract, field),
+          description: getDescription(field, contract.address, config),
```

Comment:
Looking at getValues here it seems to me this order is more consistent
```suggestion
          values: getValues(discovery, contract, field),
          description: getDescription(config, contract.address, field),
```